File: OC/OC本质/day18-MVVM/day18-MVC-Apple/ViewController.m
//
//  ViewController.m
//  day18-MVC-Apple
//
//  Created by fgy on 2019/8/5.
//  Copyright © 2019 test. All rights reserved.
//

#import "ViewController.h"
#import "FYAppleView.h"
#import "FYItemModel.h"
#import "FYNewsViewModel.h"

@interface ViewController ()
@property (nonatomic,strong) FYNewsViewModel *viewmodel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
	
	//MVP
	self.viewmodel = [[FYNewsViewModel alloc]initWithController:self];
}

@end

File: Swift/Swift_demo_01/Swift_demo_01/ViewController.swift
//
//  ViewController.swift
//  Swift_demo_01
//
//  Created by fgy on 2019/4/24.
//  Copyright © 2019 test. All rights reserved.
//

import UIKit
class SimpleClass {
    var value: Int = 0
}

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        print("file:\(#file),line:\(#line) col:\(#column) func:\(#function)")
    }


}

